bin/all_sky_search/pycbc_bin_trigger_rates_dq

```diff
@@ -75,6 +71,19 @@ with HFile(args.trig_file, 'r') as trig_file:
     data_mask = np.zeros(n_triggers_orig, dtype=bool)
     data_mask[idx] = True
 
+    # also get the gated times while we have the file open
+    gate_times = []
+    if args.gating_windows:
+        logging.info('Getting gated times')
+        try:
+            gating_key = f'{ifo}/gating'
```

Comment:
I'm not seeing why the variable `gating key` is needed, does something bad happen if you just go directly to `gating_types = trig_file[f'{ifo}/gating'].keys()` ?